### DISCUSSION TITLE
Docker lookup for artifactory fails with Mend Github App

### DISCUSSION THREAD

**Discussion by simone-dandreta** [renovatebot/renovate#21788] [Request Help] [answered]
### I'm using the Mend Renovate hosted app on github.com.

- [X] I'm using the Mend Renovate hosted app.

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe your problem.

This is my renovate.json file:

```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "hostRules": [
    {
      "matchHost": "myartifactory.corp",
      "hostType": "docker",
      "username": "REDACTED",
      "encrypted": {
        "password": "REDACTED"
      }
      
    }
  ]
}
```

No matter if I put https in front of the host, or tried with plain username/password, the PR always returns:

```
DEBUG: Failed to look up docker package 
```

IMPORTANT: Please note that the same configuration works if I use renovate cli locally. Just by passing the same json file and running renovate with the same target repo, it works. 
I guess this is something related to the Github App.

### Relevant debug logs.

<details><summary>Logs</summary>

</details>

**Comment by viceice** [comment]
1. properly format logs
2. post the http stats log message, so we can see which urls where called

**Reply by simone-dandreta** [reply]
@viceice I think I edited it correctly now. Please let me know, thanks!

**Reply by viceice** [reply]
i wonder why there are no http stats for the docker registry 🤔

**Reply by simone-dandreta** [reply]
Nevermind, I think the issue is that we can't use github runners with our artifactory as it's internal.

**Reply by viceice** [reply]
you can, but you need a self hosted runner which is running inside your network 😏

**Reply by simone-dandreta** [reply]
And how do I configure the Mend Renovate GH App to use my internal runners? We already have the infra ready for it, so I am wondering how I can achieve it?

**Reply by viceice** [reply]
you need to self-host renovate, you can use our official renovate action

**Reply by simone-dandreta** [reply, marked as answer]
Thanks @viceice, that's what exactly what I am working on right now. Made a good progress, connectivity to artifactory works.
I think this discussion can be closed. Thanks!

**Comment by simone-dandreta** [comment]
@viceice is it possible for you to remove permanently this discussion? it contains our URL (in the editing history) and we'd like to remove it for security reasons. Thank you

**Reply by HonkingGoose** [reply]
I think you can delete the URL in the edit history yourself. Read the [GitHub Docs, Deleting sensitive information from a comment's history](https://docs.github.com/en/communities/moderating-comments-and-conversations/tracking-changes-in-a-comment#deleting-sensitive-information-from-a-comments-history) to learn how.

If you still need help after reading the docs above, let us know.

**Reply by simone-dandreta** [reply]
I think I did it, thanks!
